libcloud/compute/base.py

```diff
@@ -447,7 +447,7 @@ class NodeLocation(object):
     'US'
     """
 
-    def __init__(self, id, name, country, driver):
+    def __init__(self, id, name, country, driver, extra=None):
```

Comment:
This change is breaking GCE driver because it calls parent constructor after setting ``extra`` attribute on the class instance.

I have a fix locally which I will include when merging those changes.